script.js

```diff
@@ -52,11 +52,47 @@ document.getElementById('book-form').addEventListener('submit', (e) => {
   Books.display(title, author);
   Books.addBookToStorage(title, author);
   document.getElementById('book-form').reset();
+  window.confirm('Book is added');
 });
 
 const dbtn = document.querySelector('#bookContainer');
 dbtn.addEventListener('click', (e) => {
   const title = e.target.parentNode.firstChild.nextSibling.firstChild.nextSibling;
   const book = new Books();
   book.removeBook(title.textContent, title.parentNode);
-});
\ No newline at end of file
+});
+
+const listbtn = document.getElementById('list-container');
+const addbtn = document.getElementById('book-form');
+const contactbtn = document.getElementById('contact-container');
+
+document.getElementById('list-btn').addEventListener('click', () => {
+  listbtn.classList.remove('hide');
+  addbtn.classList.add('hide');
+  contactbtn.classList.add('hide');
+});
+
+document.getElementById('add-btn').addEventListener('click', () => {
+  listbtn.classList.add('hide');
+  addbtn.classList.remove('hide');
+  contactbtn.classList.add('hide');
+});
+
+document.getElementById('contact-btn').addEventListener('click', () => {
+  listbtn.classList.add('hide');
+  addbtn.classList.add('hide');
+  contactbtn.classList.remove('hide');
+});
+
+function printTime() {
+  const currentDate = document.getElementById('currentDate');
+  const d = new Date();
```

Comment:
- [ ] **Please use the Luxon library to handle the date.**

As a part of the requirements for this project, you have to use the Luxon library for the date. 

- Display the current date below the navigation bar. For this requirement, you should use a library - [Luxon](https://moment.github.io/luxon/#/install).
  - See the article [Get the current date and time](https://moment.github.io/luxon/#/tour?id=get-the-current-date-and-time).